pandas/plotting/_core.py

```diff
@@ -2951,21 +2951,46 @@ def box(self, by=None, **kwds):
 
     def hist(self, by=None, bins=10, **kwds):
         """
-        Histogram
+        Draw one histogram of the DataFrame's columns.
+
+        A histogram is a representation of the distribution of data.
+        This function groups the values of all given Series in the DataFrame
+        into bins, and draws all bins in only one matplotlib.AxesSubplot. This
+        is useful when the DataFrame's Series are in a similar scale.
 
         Parameters
         ----------
-        by : string or sequence
+        by : str or sequence, optional
             Column in the DataFrame to group by.
-        bins: integer, default 10
-            Number of histogram bins to be used
-        `**kwds` : optional
-            Additional keyword arguments are documented in
-            :meth:`pandas.DataFrame.plot`.
+        bins : int, default 10
+            Number of histogram bins to be used.
+        **kwds : optional
+            Keyword arguments to pass on to :meth:`pandas.DataFrame.plot`.
```

Comment:
Can you keep the original sentence here? (it was originally how you have it now, but we changed it in one of the previous PRs to say that ""Additional keywords are documented in DataFrame.plot"

Reply:
@jorisvandenbossche We will have to revise this after all the sprint PRs have been merged, since people have been changing this and it might not be consistent anymore. But it will be fast and easy.

See https://github.com/pandas-dev/pandas/issues/20348